py/selenium/webdriver/ie/webdriver.py

```diff
@@ -45,19 +45,20 @@ def __init__(self, executable_path='IEDriverServer.exe', capabilities=None,
          - capabilities: capabilities Dictionary object
          - port - port you would like the service to run, if left as 0, a free port will be found.
          - log_level - log level you would like the service to run.
-         - log_file - log file you would like the service to log to.
+         - service_log_path - target of logging of service, may be "stdout", "stderr" or file path.
          - options: IE Options instance, providing additional IE options
          - desired_capabilities: alias of capabilities; this will make the signature consistent with RemoteWebDriver.
         """
+        if log_file:
+            warnings.warn('use service_log_path instead of log_file', DeprecationWarning)
+            service_log_path = log_file
         if ie_options:
             warnings.warn('use options instead of ie_options', DeprecationWarning)
             options = ie_options
         self.port = port
         if self.port == 0:
             self.port = utils.free_port()
         self.host = host
-        self.log_level = log_level
-        self.log_file = log_file
```

Comment:
I don't see a code relevance as to why they are included as instance vars other than the fact that you can check them after setting them since they aren't available on the `Service`

Reply:
But you're right, I don't see them in any other webdrivers.